zeek/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# CHANGELOG - zeek
+
```

Comment:
Will this be released the same way as other integrations in the future? If yes, we need a towncrier header here.

Reply:
Yes